app/code/Magento/Shipping/Test/Unit/Controller/Adminhtml/Order/ShipmentLoaderTest.php

```diff
@@ -100,8 +100,8 @@ protected function setUp()
             'shipment_id' => 1000065,
             'shipment' => ['items' => [1 => 1, 2 => 2]],
             'tracking' => [
-                ['number' => 'jds0395', 'title' => 'DHL', 'carrier_code' => 'dhl'],
-                ['number' => 'lsk984g', 'title' => 'UPS', 'carrier_code' => 'ups'],
+                ['number' => 'jds0395', 'title' => 'DHL', 'carrier_code' => 'dhl', 'track_url' => ''],
```

Comment:
Looks like this test does not test the feature which is added in this PR, as track url may be valid or not.